code/modules/mob/living/carbon/human/species/station/traits_vr/neutral.dm

```diff
@@ -33,6 +33,16 @@
 	var_changes = list("cold_discomfort_level" = T0C+21)
 	excludes = list(/datum/trait/cold_discomfort)
 
+/datum/trait/winged_flight
+	name = "Winged Flight"
+	desc = "Allows you to fly by using your wings."
+	cost = 1
```

Comment:
the traits system was not designed to allow costs on neutral traits. This cost does nothing. All neutral traits are free regardless of what you put here. If it has a cost, it's a positive trait. If it gives cost back, it's a negative trait.

Also just, nah imo. The limitations on flight were added due to discussion on the original flight PR. "Wait a year then remove them" isn't powercreep at all nuh uh.

Why don't you fix the actual problems you're complaining about instead of just "eh make it half the cost and not take up a trait slot" which is a lazyass way to improve quality of life. Improve the quality, don't make the cost of living poorly lower. ;x

Reply:
First off, correct, price reduction and neutral traits aren't exactly compatible. I'll move the trait back into positive as it should still cost something.
And then secondly, i am trying to fix the actual problems of the perk. Starting with the price balance which is the easiest and the most important one to to me. Then maybe i will give time to 'fix actual problems' so this perk is becomes good enough to be worth investing 2 points into it. An hack fix with strong potential for a true fix later on is much better than no fix at all.